JuiceMaker/JuiceMaker/Controller/StockViewController.swift

```diff
@@ -7,9 +7,86 @@
 
 import UIKit
 
-final class StockViewController: UIViewController {
+protocol sendStockDelegate {
+    func sendStock() -> JuiceMaker
+}
+
+final class StockViewController: UIViewController{
+    var delegate: sendStockDelegate?
```

Comment:
```suggestion
    weak var delegate: sendStockDelegate?
```

를 사용하여 래퍼런스 카운트를 올리지 않는 방법이 좋습니다.

Reply:
protocol에 AnyObject추가 후 weak 까지 추가 하였습니다!